src/orchestrator/orchestrator/orchestrator.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import codecs
+import datetime
+import logging
+from time import sleep
+import jsonpickle
+
+from learner.learner import learn_for_users
+from scraper.scraper import scrap
+from server.frontendstructs import Document
+from topicmodeller.topicmodeller import TopicModeller, classify
+
+
+logging.basicConfig(format=u'%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO, ilename='orchestrator.log')
+
+
+def _to_json(document):
+    # set unicode and pretty-print
+    jsonpickle.set_encoder_options('simplejson', indent=4, ensure_ascii=False)
+    return jsonpickle.encode(document)
+
+
+def _dump_json_document(document, folder):
+    json = _to_json(document)
+    filename = folder + '/' + str(datetime.datetime.utcnow()) + '.json'
+    with codecs.open(filename=filename, mode='w', encoding='utf-8') as file_desc:
+        file_desc.write(json)
+
```

Comment:
the name "dump_json_document" let me thing the document parameter is already json, maybe _dump_document_as_json(document, destination_folder). Yes I'm sure this method has existed for month but never shocked me :-)